php/media/class-filter.php

```diff
@@ -734,6 +734,23 @@ public function init_rest_filters() {
 		}
 	}
 
+	/**
+	 * Record attachment with meta being updated.
+	 *
+	 * @hook wp_update_attachment_metadata
+	 *
+	 * @param array $data The new meta array.
+	 * @param int   $id   The id.
+	 *
+	 * @return array
+	 */
+	public function record_meta_update( $data, $id ) {
+		$this->media->plugin->settings->set_param( '_currrent_attachment', $id );
+		$this->media->plugin->settings->set_param( '_currrent_meta', $data );
```

Comment:
@DavidCramer 

What's the use case for this param `_currrent_meta`?

Reply:
None yet, but doesn't hurt for a future requirement...